hera_sim/visibilities/healvis_wrapper.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import division
 
-import healpy
+from astropy_healpix import healpy
```

Comment:
I can't comment on the lines below that use this, but the healpy compatible interface is not the recommended interface.